trade-finance-manager-api/src/v1/api-response-types/utilisation-report-reconciliation-details-response-body.ts

```diff
@@ -1,6 +1,6 @@
 import { IsoDateTimeStamp, ReportPeriod, UtilisationReportReconciliationStatus, FeeRecordStatus, CurrencyAndAmount } from '@ukef/dtfs2-common';
 
-type FeeRecordItem = {
+export type FeeRecordItem = {
```

Comment:
Can we move this to a common file since it is used in multiple responses now please?

Reply:
I'm also going to rename this to `FeeRecord` as the `Item` part feels a bit redundant now that we have more clear definitions around things like fee record payment groups